pkg/asset/internal/templates.go

```diff
@@ -233,8 +233,8 @@ spec:
           readOnly: false
         command:
         - "/bin/sh"
-        - "-c"
-        - "/usr/bin/flock --exclusive --timeout=30 /var/lock/kenc.lock; kenc -r -m iptables && kenc -m iptables"
+        - "-ec"
```

Comment:
Aso doesn't seem like this needs to be wrapped in a shell. Can't we just execute flock directly?